### DISCUSSION TITLE
Is there a Scale to Fit option?

### DISCUSSION THREAD

**Discussion by alcor-sasho** [ClosedXML/ClosedXML#2107] [Q&A] [answered]
Hello, 

I would like to set the Scale to Fit option of the width to always be 1 page, so that when printing, all columns (regardless of the number) are on 1 page. Is there such a parameter? 

Thanks in advance!

Alex

**Comment by jahav** [comment, marked as answer]
`IXLWorksheet.PageSetup.FitToPages()` is likely what you are looking for (never used this myself).  [ClosedXML.Examples/PageSetup](https://github.com/ClosedXML/ClosedXML/tree/develop/ClosedXML.Examples/PageSetup) has some examples, the [Page](https://github.com/ClosedXML/ClosedXML/blob/develop/ClosedXML.Examples/PageSetup/Page.cs) one likely what you are looking for.
